src/ifcparse/CMakeLists.txt

```diff
@@ -35,12 +35,14 @@ add_library(IfcParse ${IFCPARSE_FILES})
 set_target_properties(IfcParse PROPERTIES COMPILE_FLAGS -DIFC_PARSE_EXPORTS VERSION "${PROJECT_VERSION}" SOVERSION "${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}")
 
 if(IFCXML_SUPPORT)
+    find_package(LibXml2 REQUIRED)
+    add_definitions(-DWITH_IFCXML)
+    set(SWIG_DEFINES ${SWIG_DEFINES} -DWITH_IFCXML)
```

Comment:
`SWIG_DEFINES` will be modified only locally for this cmake module, but it's intended to be used in `src/ifcwrap`. So `PARENT_SCOPE` is needed.